source/opt/folding_rules.cpp

```diff
@@ -344,6 +542,14 @@ spvtools::opt::FoldingRules::FoldingRules() {
   rules_[SpvOpPhi].push_back(RedundantPhi());
 
   rules_[SpvOpSelect].push_back(RedundantSelect());
+
+  rules_[SpvOpFAdd].push_back(RedundantFAdd());
+  rules_[SpvOpFSub].push_back(RedundantFSub());
+  rules_[SpvOpFMul].push_back(RedundantFMul());
+  rules_[SpvOpFDiv].push_back(RedundantFDiv());
+
+  rules_[SpvOpExtInst].push_back(RedundantFMix());
```

Comment:
This section is going to get pretty large.  It might be easier if we organize the rules in alphabetical order based on the opcode.